components/Lobby/Lobby.tsx

```diff
@@ -1,16 +1,33 @@
-import React, { FunctionComponent } from "react";
+import React, { FunctionComponent, useState } from "react";
 import SettingsArea from "./SettingsArea";
 import Teams from "../Teams/Teams";
 import HomeButton from "../Lobby/HomeButton";
+import PopUp from "../PopUp";
+import useRoomActions from "hooks/useRoomActions";
 
 interface Props {}
 
 const Lobby: FunctionComponent<Props> = ({}) => {
+  const [clicked, setClicked] = useState(false);
```

Comment:
Let's give those a useful name, what do these values represent not on what changed them. In this case maybe popupVisible or showPopup?